framework/src/org/checkerframework/common/util/report/qual/ReportUnqualified.java

```diff
@@ -0,0 +1,24 @@
+package org.checkerframework.common.util.report.qual;
+
+import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
+import org.checkerframework.framework.qual.InvisibleQualifier;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * An annotation intended solely for representing an unqualified type in
+ * the qualifier hierarchy for the Report Checker
+ * <p>
+ * This annotation may not be written in source code; it is an
+ * implementation detail of the checker.
+ * <p>
+ * Note that because of the missing RetentionPolicy, the qualifier will
+ * not be stored in bytecode. // TODO: set it to store in source
```

Comment:
If this is just a note for you, I would prefer you remove it.  That way it won't confuse others.